src/nrniv/multisend.cpp

```diff
@@ -576,13 +557,14 @@ void bgp_dma_receive(NrnThread* nt) {
     w2 = nrnmpi_wtime();
 #if TBUFSIZE
     tbuf_[itbuf_++] = (unsigned long) ncons;
-    tbuf_[itbuf_++] = (unsigned long) bgp_receive_buffer[current_rbuf]->nsend_cell_;
+    tbuf_[itbuf_++] = (unsigned long) multisend_receive_buffer[current_rbuf]->nsend_cell_;
     tbuf_[itbuf_++] = (unsigned long) s;
     tbuf_[itbuf_++] = (unsigned long) r;
     tbuf_[itbuf_++] = (unsigned long) dmasend_time_;
     if (use_phase2_) {
-        tbuf_[itbuf_++] = (unsigned long) bgp_receive_buffer[current_rbuf]->phase2_nsend_cell_;
-        tbuf_[itbuf_++] = (unsigned long) bgp_receive_buffer[current_rbuf]->phase2_nsend_;
+        tbuf_[itbuf_++] =
+            (unsigned long) multisend_receive_buffer[current_rbuf]->phase2_nsend_cell_;
+        tbuf_[itbuf_++] = (unsigned long) multisend_receive_buffer[current_rbuf]->phase2_nsend_;
     }
 #endif
 #if (BGPMDA & 2) && MAXNCONS
```

Comment:
BGPDMA?!?

Reply:
In fact this is BGPMDA xD I think you can delete it, it has never been executed ^^